### ISSUE TITLE
Gmail issue/pull request status

### ISSUE THREAD

**Issue by Mottie** [isaacs/github#401]
Submitted to GitHub:

> I really like the [view buttons](https://github.com/blog/1891-view-issue-pull-request-buttons-for-gmail) I get in my gmail, but it's hard to keep track of which issues are open or closed, or if pull request has been merged, not merged or closed.

> I would like to request an enhancement to add an issue status in my email. It looks like it might be possible to use something similar to the [parcel delivery](https://developers.google.com/gmail/markup/reference/parcel-delivery) method to accomplish this request.

**Comment by jglovier** [comment]
This is a really interesting idea, and agree that it would be helpful. I was under the impression that it was not possible with Gmail buttons existing support, but that parcel delivery method looks interesting.

Unfortunately, after a bit of digging, it appears that parcel delivery method is pretty tightly scoped to exactly what is intended for, and wouldn't support a use case like open or closed GitHub issues.

**Comment by Mottie** [comment]
Maybe adding a [Schema.org proposal](https://developers.google.com/gmail/markup/reference/schema-org-proposals) for this idea might help it along?

**Comment by jglovier** [comment]
>Maybe adding a Schema.org proposal for this idea might help it along?

Yeah, that's a great idea. I'm sure adding something like that to Schema wouldn't just benefit GitHub and GH users, but could be universal enough to benefit many types of services with bug/issue trackers. :sparkles: :+1:

**Comment by Mottie** [comment]
Added: http://lists.w3.org/Archives/Public/public-vocabs/2015Jun/0000.html

**Comment by jglovier** [comment]
@Mottie :heart: :metal: :zap:
